### ISSUE TITLE
mrclay/minify library: deprecation warning

### ISSUE THREAD

**Issue by getdatakick** [thirtybees/thirtybees#1493]
```
[DEPRECATION] ord(): Passing null to parameter 1 ($character) of type string is deprecated in file vendor/mrclay/minify/min/lib/JSMin.php at line 333.
```

**Comment by getdatakick** [comment]
Proper solution will be to move dependency on `mrclay/minify` library to library module. Similar to excel import functionality

**Comment by getdatakick** [comment]
Deprecation warning fixed by commit 42f08dcb8bc0dd06bc10fcbe091d388cd3e949fe

**Comment by getdatakick** [comment]
Moving of minification functionality will be done in issue #1572
